src/Strategy/Maven.hs

```diff
@@ -0,0 +1,48 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Strategy.Maven
+  ( discover',
+  )
+where
+
+import Control.Effect.Diagnostics
+import Control.Effect.Lift
+import Control.Monad.IO.Class
+import Effect.Exec
+import Effect.ReadFS
+import Graphing (Graphing)
+import Path
+import qualified Strategy.Maven.PluginStrategy as Plugin
+import qualified Strategy.Maven.Pom as Pom
+import qualified Strategy.Maven.Pom.Closure as Pom
```

Comment:
These imports have the same name, we could probably rename the second to `PomClosure`.